cmake/AddFdbTest.cmake

```diff
@@ -277,7 +277,51 @@ function(package_bindingtester)
     COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_SOURCE_DIR}/bindings ${CMAKE_BINARY_DIR}/bindingtester/tests
     COMMAND ${CMAKE_COMMAND} -E touch "${CMAKE_BINARY_DIR}/bindingtester.touch"
     COMMENT "Copy test files for bindingtester")
-  add_custom_target(copy_bindingtester_binaries DEPENDS ${outfiles}  "${CMAKE_BINARY_DIR}/bindingtester.touch")
+
+  add_custom_target(copy_binding_output_files DEPENDS ${CMAKE_BINARY_DIR}/bindingtester.touch python_binding fdb_flow_tester)
+  add_custom_command(
+    TARGET copy_binding_output_files
+    COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:fdb_flow_tester> ${bdir}/tests/flow/bin/fdb_flow_tester
+    COMMENT "Copy Flow tester for bindingtester")
+
+  set(generated_binding_files python/fdb/fdboptions.py)
```

Comment:
I think we might now be testing Ruby at the moment, but it probably also has a file that we should copy.

Reply:
Ruby is currently generating the `fdbtoptions.rb` in source directory[1] and, this script already copied the source directory. Hence things work out for Ruby.

I'm not sure how target dependency for on Ruby worked out, but it has not ever compiled out of order. @mpilman Are `vexilographer` commands executed in the order they appear on source? Or should I add ruby dependency to `bindingtester.touch` 

[1] https://github.com/apple/foundationdb/blob/master/bindings/ruby/CMakeLists.txt#L3

Reply:
Interesting, any idea why Ruby is written differently? Is what it's doing now to generate files in the source directory what we want?

Reply:
I don't have much idea except the comment in build file which says that `it simplifies debugging`. I don't see any reason why it shouldn't do what other bindings do though (as one can point to package directory to Ruby one, like in case of PYTHONPATH).

I think in general with CMake we want to keep source directory clean, so I don't think its what we want down the line.

Reply:
Ok, I guess that's a slightly separate issue, so we can proceed with this for now and worry about Ruby later.